kit/azure/organization-hierarchy/main.tf

```diff
@@ -1,3 +1,8 @@
+locals {
+  default_location  = var.location[0]
+  allowed_locations = join(",", var.location)
```

Comment:
d: consider doing the join in the tmpl.json instead (as that's the only place that's concerned with taking a list into a json string). 

also: consider using the jsonencode function instead